src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Adjournment.java

```diff
@@ -44,12 +45,12 @@ public class Adjournment {
     private DynamicList nextHearingVenueSelected;
     @JsonProperty("adjournCasePanelMembersExcluded")
     private AdjournCasePanelMembersExcluded panelMembersExcluded;
-    @JsonProperty("adjournCaseDisabilityQualifiedPanelMemberName")
-    private String disabilityQualifiedPanelMemberName;
-    @JsonProperty("adjournCaseMedicallyQualifiedPanelMemberName")
-    private String medicallyQualifiedPanelMemberName;
-    @JsonProperty("adjournCaseOtherPanelMemberName")
-    private String otherPanelMemberName;
+    @JsonProperty("adjournCasePanelMember1")
+    private JudicialUserBase panelMember1;
```

Comment:
when you use field names as exact json names you dont have to use @JsonProperty
so instead of **panelMember1** use **adjournCasePanelMember1** etc

Reply:
**Longer != worse** remember that! Always!
as for the clarification, you are right although the design kinda sucks here because if you would go for json the naming here would be shoite aswell:
```
{
 adjournment: {
   adjournCasePanelMember1:{},
   adjournCasePanelMember2:{},
   ...
 }
}
```
now when you accesing it for example on frontend **adjournment.adjournmentCasePanelMember1.....**
So even tho from OOP perspective class construction is right, from response body its not, which shows 2 issues, naming kinda sucks and inconsistency.

We are not gona make all those changes ofc, but keep in mind that the data in web services are shared across, and when providing API consistency is super duper important